avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -263,11 +270,16 @@ public Object convert(Schema schema, Object value) {
 
   private Cache<Schema, org.apache.avro.Schema> fromConnectSchemaCache;
   private Cache<org.apache.avro.Schema, Schema> toConnectSchemaCache;
-
+  private boolean includeConnectMeta;
 
   public AvroData(int cacheSize) {
+    this(cacheSize, true);
+  }
+
+  public AvroData(int cacheSize, boolean includeConnectMeta) {
```

Comment:
Nope, don't feel strongly about it. Strictly speaking, `AvroData` is not even public API so ideally people shouldn't rely on these constructors (or accept that we may sometimes change them). `AvroConverter` is the only public API this module provides.